legacy/build-deploy-docker-compose.sh

```diff
@@ -1918,8 +1918,9 @@ TLS_FALSE_INGRESSES=$(kubectl -n ${NAMESPACE} get ingress -o json | jq -r '.item
 for TLS_FALSE_INGRESS in $TLS_FALSE_INGRESSES; do
   TLS_SECRETS=$(kubectl -n ${NAMESPACE} get ingress ${TLS_FALSE_INGRESS} -o json | jq -r '.spec.tls[]?.secretName')
   for TLS_SECRET in $TLS_SECRETS; do
+    echo ">> Cleaning up certificate for ${TLS_SECRET} as tls-acme is set to false"
```

Comment:
This doesn't check whether the certificate is actually a `tls-acme`/`LE` certificate. I think this could lead into issues where non `tls-acme`/`LE` certificate gets deleted just because `kubernetes.io/tls-acme` is set to `false`.

Reply:
Do you happen to have a snippet that could verify this?

Reply:
@shreddedbacon Could we check for a `cert-manager` annotation on the secret first e.g. this ones: 
```text
   cert-manager.io/issuer-group: cert-manager.io
   cert-manager.io/issuer-kind: ClusterIssuer
   cert-manager.io/issuer-name: letsencrypt-prod
```

Reply:
I've already updated the PR to check the issuer directly from the cert using openssl.

Unfortunately, we can't rely on the `letsencrypt-prod` issuer always being called that. Numerous articles online I have seen them tell people to call it `letsencryt-production` or `prod-letsencrypt`. While for AIO cloud we know it is `letsencrypt-prod`, not everyone calls it this.